src/App.css

```diff
@@ -20,3 +20,17 @@
 .App-link {
   color: #09d3ac;
 }
+
+.button {
+  height: 50px;
+  padding: 5px 15px;
+  border-radius: 5px;
+  font-size: 1em;
+}
+.selected {
+  background-color: #282c34;
+  color: #09d3ac;
+  width: 25px;
+  height: 25px;
+  border: ;
```

Comment:
This border property isn't used.

Reply:
I updated the button styles and got rid of the extra code.